bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -274,9 +317,9 @@ public Void call() throws Exception {
             return null;
           }
         });
-        clientAuthInterceptor = oauth2Interceptor;
+        this.clientAuthInterceptor = oauth2Interceptor;
       } else {
-        clientAuthInterceptor = new ClientAuthInterceptor(credentials, batchPool);
+        this.clientAuthInterceptor = new ClientAuthInterceptor(credentials, batchPool);
         clientAuthInterceptorFuture = batchPool.submit(new Callable<Void>() {
```

Comment:
I don't know if we should call it clientAuthInterceptorFuture, it is used to refresh credentials.